storage/innobase/xtrabackup/src/xbcloud/s3.cc

```diff
@@ -712,6 +699,17 @@ bool S3_client::list_objects_with_prefix(const std::string &bucket,
             my_progname);
         return false;
       }
+      if (storage_class != "" && !storage_class_message_sent) {
```

Comment:
Please add a note about this backport on commit message

Reply:
added